test/e2e/app-dir/app-static/app/force-static/[slug]/page.js

```diff
@@ -1,17 +1,24 @@
 // force-static should override the `headers()` usage
+
+import { Suspense } from 'react'
+
 // in parent layout
 export const dynamic = 'force-static'
 
 export function generateStaticParams() {
   return [{ slug: 'first' }, { slug: 'second' }]
 }
 
+function Dynamic({ params }) {
+  return <p id="params">{JSON.stringify(params)}</p>
+}
```

Comment:
I'm curious if we should be updating existing tests like this or instead move this to a new test since it's a change in behavior

Reply:
This is special case for PPR and static generation, so I don't think we need to apply this pattern broadly.